docs/relations_guide.md

```diff
@@ -1,123 +1,255 @@
+
+For example:
+
+‘[lymph node marginal reticular cell](http://purl.obolibrary.org/obo/CL_0009103)’ subClassOf [**'located in'**](http://purl.obolibrary.org/obo/RO_0001025) *some* ['subcapsular sinus of lymph node'](http://purl.obolibrary.org/obo/UBERON_0005463)
+
+### overlaps
+
+[**'part of'**](http://purl.obolibrary.org/obo/BFO_0000050) applies in cases where an entire cell is within an anatomical structure, but some cells have parts in multiple anatomical structures. For example, many neurons span multiple regions of the central nervous system. The general relation for this is [**overlaps**](http://purl.obolibrary.org/obo/RO_0002131) (has some part in).
+
+[**overlaps**](http://purl.obolibrary.org/obo/RO_0002131) is not currently used directly in the cell ontology (time of writing 05/2023), but more specific relationships exist for recording the location of neurons and their parts. These are described in the next section.
+
+### Recording the location of neurons 
+
+### [<u>has soma location</u>](http://purl.obolibrary.org/obo/RO_0002100)
+
+When neurobiologists talk about the location of vertebrate neurons, they are typically referring to soma location. The importance of soma location to identify is underscored by how commonly cell types are named, in part, by soma location. We therefore have a dedicated relation for recording this: [**'has soma location'**](http://purl.obolibrary.org/obo/RO_0002100).
+
+For example, [anterior horn motor neuron](http://purl.obolibrary.org/obo/CL_2000048) has the following subclass axiom:
 
 [**'has soma location'**](http://purl.obolibrary.org/obo/RO_0002100) *some* ['ventral horn of spinal cord'](http://purl.obolibrary.org/obo/UBERON_0002257)
 
-In Manchester OWL syntax, this is represented as:
+**axiomatization of ‘has soma location’**
+
+-   *subPropertyOf*: overlaps \# if X has\_soma\_location some Y, then X overlaps some Y)
+
+-   *domain*: neuron \# X has\_soma\_location some Y => X is inferred to be a subClassOf neuron
+
+-   *property chain*: has\_soma\_location o part\_of --> has\_soma\_location \# If x has soma location y and y is part\_of z, then x has\_soma\_location\_z
+
+**Example of reasoning with the property chain:**
+
+'cortical interneuron' equivalentTo 'interneuron' that has\_soma\_location some 'cerebral cortex' 'rosehip neuron'
+
+'rosehip neuron' subClassOf interneuron  and has\_soma\_location some 'cortical layer 1' 
+
+'cortical layer 1'  subClassOf part\_of some 'cerebral cortex
+
+=> 'rosehip neuron' subClassOf 'cortical interneuron'
+
+### sends synaptic output to region
+
+A relationship between a neuron and a region, where the neuron has a functionally relevant number of output synapses in that region.
+
+'[<u>adult basket subesophageal neuron</u>](http://purl.obolibrary.org/obo/FBbt_00051856)' SubClassOf [<u>sends synaptic output to region</u>](https://www.ebi.ac.uk/ols4/ontologies/fbbt/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FRO_0013003?lang=en)
```

Comment:
Is there a particular reason for using an OLS4-specific URL, instead of the standard PURL-based IRI `http://purl.obolibrary.org/obo/RO_0013003`?

Reply:
Inherited from some older version of doc.  Agree we should just use PURLs.

Reply:
Have updated to PURLs only [5cd8021](https://github.com/obophenotype/cell-ontology/commit/5cd80216a7dd13372a649354ab38bc56fcc0d97a)